code/game/objects/items/weapons/RPD.dm

```diff
@@ -529,6 +529,9 @@ var/global/list/RPD_recipes=list(
 
 	if(istype(A,/area/shuttle)||istype(A,/turf/space/transit))
 		return 0
+	
+	if(is_anchored_dense_turf(get_turf(A))) // This used to be allowed to help people rebuild pipes destroyed under walls etc
+		return 0 // But some people exploited it and forced me to do this
```

Comment:
duncathan I don't see that feature used to much when rebuilding, as the resulting pipes end up layared above the wall.